fly/commands/ordering_instanced_pipeline.go

```diff
@@ -0,0 +1,58 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/concourse/concourse/atc"
+	"github.com/concourse/concourse/go-concourse/concourse"
+
+	"github.com/concourse/concourse/fly/commands/internal/displayhelpers"
+	"github.com/concourse/concourse/fly/commands/internal/flaghelpers"
+	"github.com/concourse/concourse/fly/rc"
+)
+
+type OrderInstancedPipelinesCommand struct {
+	Pipeline     string                         `short:"g" long:"group" required:"true" description:"Name of the instance group"`
+	InstanceVars []flaghelpers.InstanceVarsFlag `short:"p" long:"pipeline" required:"true" description:"Instance vars identifying pipeline (can be specified multiple times to provide relative ordering)"`
```

Comment:
Was there a reason we wanted to have the flag name be `"pipeline"` instead of `"instance_var"` or `"i"` for short? I'm asking because for the `set-pipeline` command we use `"instance-var"` as the flag name where you pass in the instance vars that are to be set https://github.com/concourse/concourse/blob/34474b0f0cd5b6a46639b517e4b38d3d4250f6e7/fly/commands/set_pipeline.go#L29
With how the `set-pipeline` command is used, I thought that a `pipeline` can have multiple `instances` but for this `order-instance-pipelines` command it looks like a `group` (equivalent to `pipeline`) can have multiple `pipelines` (equivalent to `instances`). Just wondering what your thoughts are!

Reply:
We tried to leave the call to this command as similar as possible to the [order-pipeline](https://concourse-ci.org/managing-pipelines.html#fly-order-pipelines) command (so I could be friendly for the user), that's why we left the `pipeline` name flag to identify each instance pipeline and we added another flag called `group` to identify pipeline.

We checked also the syntax of the [set-pipeline](https://concourse-ci.org/setting-pipelines.html#fly-set-pipeline) command, but we though it could be more confusing for the user that way, because of the way [order-pipeline](https://concourse-ci.org/managing-pipelines.html#fly-order-pipelines) command was created.

Also we didn't want it to match [set-pipeline](https://concourse-ci.org/setting-pipelines.html#fly-set-pipeline) because `-i` it's used in a different way here, `-i` in [set-pipeline](https://concourse-ci.org/setting-pipelines.html#fly-set-pipeline) always sets a single instance var using `KEY=VALUE`, while `-p` in `order-instanced-pipelines` uses `KEY1:VALUE1,KEY2:VALUE2,...`
which could introduce its own confusion, because `-p` is usually `GROUP/KEY1:VALUE1,KEY2:VALUE2,...,` while here we omit the `GROUP/` because it's redundant, but still, it's closer to `-p` in other commands than `-i` in set-pipeline, we think.

Also, you're conceptually ordering pipelines (`-p`), not ordering instance vars (`-i`)

Reply:
I see, I think I understand! I always saw instance pipelines as a subset of a pipeline, but I guess I should more be thinking about it as an instance pipeline is equal to just a pipeline and pipelines can be grouped. Thanks for the explanation!

I think a small suggestion I have is to rename the `Pipeline` field to `Group` so that it is clear that the field is referencing a group rather than a pipeline?

Reply:
That's right, I just made the change there